src/ParallelAlgorithms/DiscontinuousGalerkin/InitializeInterfaces.hpp

```diff
@@ -103,6 +109,8 @@ struct InitializeInterfaces {
       Directions,
       ::Tags::InterfaceComputeItem<Directions, ::Tags::Direction<dim>>,
       ::Tags::InterfaceComputeItem<Directions, ::Tags::InterfaceMesh<dim>>,
+      ::Tags::InterfaceComputeItem<
+          Directions, ::Tags::BoundaryCoordinates<dim, Frame::Inertial>>,
```

Comment:
My only question is why are the coordinates on the internal faces being added? The coordinates for the external faces are needed for imposing boundary conditions but I'm not familiar with a use case for the coordinates on the internal faces.

Reply:
Turns out I don't need them after all, thanks for pointing this out. Removed this.

I also removed the part where I move the exterior variables simple tag to the Evolution/DG initializer, because this needs further thought in a future PR (For systems where the metric is a variable we need it on exterior faces to compute the FaceNormal. I know what to do about it, but it's not part of this PR). Since this is a lot of removed code I squashed it directly, I think it's easier to review like this.